subsys/trusted_storage/Kconfig

```diff
@@ -158,7 +158,7 @@ choice TRUSTED_STORAGE_STORAGE_BACKEND
 
 config TRUSTED_STORAGE_STORAGE_BACKEND_SETTINGS
 	bool "Settings storage backend"
-	depends on SETTINGS_NVS || SETTINGS_ZMS
+	depends on SETTINGS_ZMS || (SETTINGS_NVS && !SOC_NRF54L15)
```

Comment:
does trusted storage need to reside in internal memory? Or can someone have it on an external SPI flash device?

Reply:
Internal memory only right now.